e2e/cases/resolve/__snapshots__/index.test.ts.snap

```diff
@@ -51,22 +51,13 @@ function __webpack_require__(moduleId) {
         return Object.prototype.hasOwnProperty.call(obj, prop);
     };
 })();
-// webpack/runtime/make_namespace_object
-(()=>{
-    // define __esModule on exports
-    __webpack_require__.r = function(exports) {
-        if ('undefined' != typeof Symbol && Symbol.toStringTag) Object.defineProperty(exports, Symbol.toStringTag, {
-            value: 'Module'
-        });
-        Object.defineProperty(exports, '__esModule', {
-            value: true
-        });
-    };
-})();
-/************************************************************************/ var __webpack_exports__ = {};
-__webpack_require__.r(__webpack_exports__);
-/* harmony import */ var _browser_false_util__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(\\"?69d8\\");
-/* harmony import */ var _browser_false_util__WEBPACK_IMPORTED_MODULE_0___default = /*#__PURE__*/ __webpack_require__.n(_browser_false_util__WEBPACK_IMPORTED_MODULE_0__);
-console.log('foo:', _browser_false_util__WEBPACK_IMPORTED_MODULE_0___default()); // the value of \`foo\` should \`{}\`
+/************************************************************************/ // CONCATENATED MODULE: ./bar.ts
+const bar = 'bar';
+// EXTERNAL MODULE: <ROOT>/e2e/cases/resolve/false/./browser-false/util (ignored)
```

Comment:
@Timeless0911 added an extra module

Reply:
Tracks the edge case in https://github.com/web-infra-dev/rslib/issues/149.